src/assets/CSSAsset.js

```diff
@@ -1,12 +1,10 @@
 const Asset = require('../Asset');
 const postcss = require('postcss');
 const valueParser = require('postcss-value-parser');
-const path = require('path');
-const md5 = require('../utils/md5');
 const postcssTransform = require('../transforms/postcss');
 const CssSyntaxError = require('postcss/lib/css-syntax-error');
 
-const URL_RE = /url\s*\(\"?(?![a-z]+:)/;
+const URL_RE = /url\s*\("?(?![a-z]+:)/;
```

Comment:
Here too. ESLint seems to have removed some bit of regex. Again, ignore if I'm wrong. 😅

Reply:
Same here. The `\` was an unnecessary character escape. Same functionality.